### ISSUE TITLE
Keyerror 'template' with proxmox 7.0-8

### ISSUE THREAD

**Issue by SpaceWrangler616** [xezpeleta/Ansible-Proxmox-inventory#41]
Not a python dev, but I upgraded to proxmox v7.0-8 and the inventory script stopped working. I'm not sure if it was this way before but I debugged in VS Code and it looks like VMs that are _not_ templates don't have the 'template' key. 

```shell
$ python3 proxmox.py --list --pretty
Traceback (most recent call last):
  File "/home/dan/src/space/home-lab/proxmox.py", line 447, in <module>
    main()
  File "/home/dan/src/space/home-lab/proxmox.py", line 432, in main
    data = main_list(options, config_path)
  File "/home/dan/src/space/home-lab/proxmox.py", line 299, in main_list
    results['all']['hosts'] += qemu_list.get_names()
  File "/home/dan/src/space/home-lab/proxmox.py", line 68, in get_names
    return [vm['name'] for vm in self if vm['template'] != 1]
  File "/home/dan/src/space/home-lab/proxmox.py", line 68, in <listcomp>
    return [vm['name'] for vm in self if vm['template'] != 1]
KeyError: 'template'
```

I did some googling and changed return in the get_names function do a .get('template') instead of trying to get a key that's not there:

```python
def get_names(self):
        if self.ver >= 4.0:
            #this is broken for VMs that aren't templates
            # return [vm['name'] for vm in self if vm['template'] != 1] 
            
            #this seemed to fix it
            return [vm['name'] for vm in self if vm.get('template') == False]
        else:
            return [vm['name'] for vm in self]
```
Now it's working again.

**Comment by SpaceWrangler616** [comment]
Oops, didn't see a PR already exists to fix it.

**Comment by xezpeleta** [comment]
Fixed in #39
